common/tools/dev-tool/src/commands/about.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license
+
+import chalk from "chalk";
+
+import { baseCommands, baseCommandInfo } from ".";
+import { resolveProject } from "../util/resolveProject";
+import { createPrinter } from "../util/printer";
+import { leafCommand } from "../util/commandBuilder";
+import { printCommandUsage } from "../util/printCommandUsage";
+
+const log = createPrinter("help");
+
+const banner = `\
+       _______        __                __              __
+      / / ___/   ____/ /__ _   __      / /_____  ____  / /
+ __  / /\\__ \\   / __  / _ \\ | / /_____/ __/ __ \\/ __ \\/ / 
+/ /_/ /___/ /  / /_/ /  __/ |/ /_____/ /_/ /_/ / /_/ / /  
+\\____//____/   \\__,_/\\___/|___/      \\__/\\____/\\____/_/   
+
+Developer quality-of-life command for the Azure SDK for JS
+`;
+
+export const commandInfo = {
+  name: "about",
+  description: "display command help and information"
+} as const;
+
+export default leafCommand(commandInfo, async (options) => {
+  console.log(chalk.blueBright(banner));
+
+  try {
+    const packageInfo = await resolveProject(__dirname);
+    console.log(chalk.blueBright(`  Name/Version:\t${packageInfo.name}@${packageInfo.version}`));
+    console.log(chalk.blueBright(`  Location:\t${packageInfo.path}`));
+    console.log();
```

Comment:
feels weird to have a mix of straight `console.log` with a custom `log` method

Reply:
Here, I don't want the command namespace prepended to each message, which we do generally want for logging messages and output from commands. This command is a little special in that regard in that it's printing straight to the console, so that's why there's the disconnect. I don't expect the output functions to be mixed anywhere else.